salt/_grains/caasp_nodename.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
This one concerns myself a bit - I am not sure if it would make sense to symlink python3 to python if no python2 is installed.

Otherwise this will always flip-flop between working on `python2` or `3`, *if* it ever gets run as a script (which i doubt will happen), so we could probably also remove the whole shebang line.

Reply:
Do this script require python3? Or could it run with python2 as well?

Reply:
I would just require `python3`, we don't have to be so flexible, right? So `env python3` seems right to me...

Reply:
We are still building images with sles12sp3 with python2, so if possible we should be compatible so we don't break the current images.

Reply:
I believe the same as @inercia - or they even just `import` it, which ignores the interpreter line completely. I'll just remove the line completely to avoid confusion.